app/utils/pages.tsx

```diff
@@ -10,26 +13,36 @@ interface IPage {
     path: string
 }
 
+export function getPathsToMap (){
```

Comment:
nit: shouldn't this have a type
```suggestion
export function getPathsToMap (): string[] {
```